buildconfig/download_win_prebuilt.py

```
@@ -193,7 +193,7 @@ def place_downloaded_prebuilts(temp_dir, move_to_dir, x86=True, x64=True):
         'prebuilt-x86'
     )
 
-    ignore = None
+    ignore = lambda directory, files: ["stdint.h"]
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)